src/Quartz.Tests.Unit/Impl/Triggers/SimpleTriggerImplTest.cs

```diff
@@ -0,0 +1,379 @@
+using NUnit.Framework;
+using Quartz.Impl.Triggers;
+using System;
+
+namespace Quartz.Tests.Unit.Impl.Triggers
+{
+    [TestFixture]
```

Comment:
nit: NUnit doesn't require `[TestFixture]` anymore, `[Test]` is enough for discovery